executable/dex.ksy

```diff
@@ -666,8 +666,7 @@ types:
       - id: insns_size
         type: u4
         doc: Size of the instructions list, in 16-bit code units.
-      - id: instructions
-        -orig-id: insns
+      - id: insns
```

Comment:
I like the former variant better

Reply:
`instructions` is a difficult word to shorten and `insns` looks like a confusing tongue twister. How about `body` and `body_size` instead?

Reply:
maybe `opcodes`, but it is not much shorter.

>I could also change the name of the code_item instance in encoded_method from method_body to just code 

Will it be confusing?

Reply:
> maybe opcodes, but it is not much shorter.

That'd be incorrect, as opcodes aren't the same as instructions. An opcode(operation code) is a piece of what makes up an instruction; it defines the operation of the instruction. With that said, it wouldn't make much sense now to name the pool of instructions to `opcodes`.

I vote for `body` and `body_size` or `inst_pool` and `inst_pool_size`.

Reply:
>That'd be incorrect, as opcodes aren't the same as instructions. An opcode(operation code) is a piece of what makes up an instruction; it defines the operation of the instruction. 

You are right. But `body` is IMHO too generic, and truncations are non-obvious.

Reply:
True, so `inst_pool` then or do you have another suggestion?

Reply:
maybe just `ops`, but it is a truncation.

Reply:
Fair enough. Thanks